app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt

```diff
@@ -126,7 +126,7 @@ class PinPasswordActivityTest {
   }
 
   @Test
-  fun testPinPasswordActivityWithAdmin_checkKeyboardIsVisibleByDefault() {
+  fun testPinPasswordActivityWithAdmin_keyboardIsVisibleByDefault() {
```

Comment:
testPinPasswordActivity_withAdmin_keyboardIsVisibleByDefault

Reply:
ditto for all

Reply:
Done for all